src/proxy/ClientTable.php

```diff
@@ -3,8 +3,10 @@
 namespace luya\admin\proxy;
 
 use Curl\Curl;
+use Symfony\Component\Debug\Exception\UndefinedMethodException;
 use Yii;
 use yii\base\BaseObject;
+use yii\db\Exception;
```

Comment:
At first sight from the commit diff, this also seems not to be used in the context.